modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -41,13 +45,17 @@ You can publish an API specification to Anypoint Exchange for use in API Manager
 
 == How API Designer Relates to Other MuleSoft Products
 
-API Designer is an integral part of the MuleSoft API-led approach to integrating applications. The following diagram depicts how the API specifications that you write in API Designer can be used in other MuleSoft products.
+API Designer is an integral part of the MuleSoft API-led approach to integrating applications. The following diagram shows how the API specifications that you write in API Designer can be used in other MuleSoft products.
 
-image::apid-system-overview.png[]
+image::apid-system-overview.png[Diagram of MuleSoft product relationships]
 
-A specification writer uses API Designer to design API specifications and RAML fragments. RAML specifications can use assets that are hosted in Exchange. The specification writer can publish iterations and final versions to Anypoint Exchange.
+The diagram shows how:
 
-From Exchange, the specification can be imported into Anypoint Studio and Flow Designer as a RESTful connector, or into API Manager. Anypoint Studio can even import a specification directly from API Designer.
+* A specification writer uses API Designer to design API specifications and RAML fragments. 
```

Comment:
Is "a specification writer" a person or a product feature?
I've never seen this phrase used to describe a person's role.